.env.example

```
@@ -47,7 +47,10 @@ SPOTIFY_REFRESH_TOKEN=""
 # ============================================================
 # YouTube Music
 # ============================================================
-YTM_AUTH_USER=""
+# Browser profile index used by YouTube Music (usually 0).
+YTM_AUTH_USER="0"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)